apps/full-site-editing/full-site-editing-plugin/donations/fetch-status.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+
+const fetchStatus = async () => {
+	try {
+		const result = await apiFetch( {
+			path: '/wpcom/v2/memberships/status',
+			method: 'GET',
+		} );
```

Comment:
Why are we wrapping this in a try catch? Feels like there isn't any special handling here.

Reply:
I just like them 😄  The `catch` will handle any network failures.